include/zephyr/posix/sys/sysconf.h

```diff
@@ -141,6 +141,13 @@ enum {
 	_SC_TZNAME_MAX,
 };
 
+/*
+ * clang-format and checkpatch disagree on formatting here, so use rely and checkpatch and disable
```

Comment:
Typo, probably should be corrected.
```suggestion
 * clang-format and checkpatch disagree on formatting here, so rely on checkpatch and disable
```